src/DefinitionGenerator.ts

```diff
@@ -144,11 +144,11 @@ export class DefinitionGenerator {
       operationObj.deprecated = true;
     }
 
-    if (operationObj.requestBody) {
+    if (documentationConfig.requestBody) {
       operationObj.requestBody = this.getRequestBodiesFromConfig(documentationConfig);
     }
 
-    if (operationObj.parameters) {
+    if (documentationConfig.parameters) {
```

Comment:
This should be `documentationConfig.queryParams`, as parameters in Swagger are names `queryParams` in this plugin.

Reply:
Hmm, thinking about this, you might have to check for the existence of _any_ type of parameter, be it path, query, header (undocumented) or cookie.